generators/assets/webpack/templates/webpack.config.js.tmpl

```diff
@@ -1,117 +1,104 @@
-var webpack = require("webpack");
-var glob = require("glob");
-var CopyWebpackPlugin = require("copy-webpack-plugin");
-var ExtractTextPlugin = require("extract-text-webpack-plugin");
-var ManifestPlugin = require("webpack-manifest-plugin");
-var PROD = process.env.NODE_ENV || "development";
-var CleanWebpackPlugin = require("clean-webpack-plugin");
+const Webpack = require("webpack");
+const Glob = require("glob");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const ManifestPlugin = require("webpack-manifest-plugin");
+const CleanWebpackPlugin = require("clean-webpack-plugin");
+const UglifyJsPlugin = require("uglifyjs-webpack-plugin");
 
-var entries = {
-  application: [
-    './node_modules/jquery-ujs/src/rails.js',
-    './assets/css/application.scss',
-  ],
-}
+const configurator = {
+  entries: function(){
+    var entries = {
+      application: [
+        './node_modules/jquery-ujs/src/rails.js',
+        './assets/css/application.scss',
+      ],
+    }
+    
+    Glob.sync("./assets/*/*.*").reduce((_, entry) => {
```

Comment:
If you use reduce here, you'll drop the first file, even if it's not application.scss. :)